tutorials/identifying-first-day-no-snow/index.md

```diff
@@ -0,0 +1,516 @@
+  // it will be accessible to the addDateBands mapped to the collection below.
+  startYear = year;
+  // Get the first day-of-year for this year as an ee.Date object.
+  var firstDoy = ee.Date.fromYMD(year, 1, 1);
+  // Advance from the firstDoy to the user-defined startDay; subtract 1 since
+  // firstDoy is already 1. Set the result as the global startDate variable so
+  // that it is accessible to the addDateBands mapped to the collection below.
+  startDate = firstDoy.advance(startDoy-1, 'day');
+  // Get endDate for this year by advancing 1 year from startDate.
+  // Need to advance an extra day because end date of filterDate() function
+  // is exclusive.
+  var endDate = startDate.advance(1, 'year').advance(1, 'day');
+  // Filter the complete collection by the start and end dates just defined.
+  var yearCol = completeCol.filterDate(startDate, endDate);
+  // Construct an image where pixels represent the first day within the date range
+  // that the lowest snow fraction is observed.
+  var noSnowImg = yearCol
+    // Add date bands to all images in this particular collection.
+    .map(addDateBands)
+    // Sort the images by time.
+    .sort('millis')
+    // Make a mosaic composed of pixels from images that represent the
+    // observation with the minimum percent snow cover (defined by the
+    // NDSI_Snow_Cover band); include all associated bands for the selected
+    // image.
+    .reduce(ee.Reducer.min(5))
+    // Rename the bands - band names were altered by previous operation.
+    .rename(['snowCover', 'calDoy', 'relDay', 'millis', 'year'])
+    // Apply the mask.
+    .updateMask(analysisMask)
+    // Set the year as a property for filtering by later.
+    .set('year', year);
+    
+  // Mask by minimum snow fraction - only include pixels that reach 0
+  // percent cover. Return the resulting image.
+  return noSnowImg.updateMask(noSnowImg.select('snowCover').eq(0));
+});
+```
+
+Convert the `ee.List` of images to an image collection.
+
+```js
+var annualCol = ee.ImageCollection.fromImages(annualList);
+```
+
+## Data summary and visualization
+
+The following are a series of examples for how to display and explore the
+first DOY with no snow dataset you just generated.
+ 
+**Note**: 
+
+- These examples refer to the calendar date (`calDoy` band) when
+displaying and incorporating date information in calculations. If you are
+using a date range that begins on any day other than January 1 (DOY = 1) you
+may want to replace `calDoy` with `relDoy` in all cases below.
+
+- Results may appear different as you zoom in and out of the map because of
+[tile aggregation](https://developers.google.com/earth-engine/scale). It is best
+to view map data interactively with a relatively high zoom level. Additionally,
+for any analysis where a function provides a scale parameter, it is best to
+define it with the native resolution of the dataset (500m), which can include
```

Comment:
Did you mean "which allows for" here?

Reply:
@gino-m can you specify the which phrase needs clarification - I'm not seeing where "which allows for" might replace something.

Reply:
Sorry, I was referring to "which can include". Assuming I misread this, to avoid confusion the phrase could be rephrased, for example:

> Additionally, for any analysis where a function provides a scale parameter (e.g., region reduction, exporting results), it is best to define it with the native resolution of the dataset (500m).

Reply:
I see, yes, that is better - thanks. Done.